src/net/sf/memoranda/PlanSummaryImpl.java

```diff
@@ -0,0 +1,226 @@
+		_el.addAttribute(new Attribute("timeCode", Integer.toString(arrayTime[2])));
+		_el.addAttribute(new Attribute("timeCompile", Integer.toString(arrayTime[3])));
+		_el.addAttribute(new Attribute("timeTest", Integer.toString(arrayTime[4])));
+		_el.addAttribute(new Attribute("timePM", Integer.toString(arrayTime[5])));
+		return _el;
+	}
+	
+	/**
+	 * 
+	 * @see net.sf.memoranda.PlanSummary#getPrjName()
+	 */
+	public String getPrjName() {
+		return _el.getAttributeValue("prjName");
+	}
+	
+	/**
+	 * 
+	 * @see net.sf.memoranda.PlanSummary#getPlanLOCperHr()
+	 */
+	public int getPlanLOCperHr() {
+		return Integer.parseInt(_el.getAttributeValue("planLOC"));
+	}
+	
+	/**
+	 * 
+	 * @see net.sf.memoranda.PlanSummary#getLOCBasePlan()
+	 */
+	public int getLOCBasePlan() {
+		return Integer.parseInt(_el.getAttributeValue("psizeBase"));
+	}
+	
+	/**
+	 * 
+	 * @see net.sf.memoranda.PlanSummary#getLOCDeletedPlan()
+	 */
+	public int getLOCDeletedPlan() {
+		return Integer.parseInt(_el.getAttributeValue("psizeDeleted"));
+	}
+	
+	/**
+	 * 
+	 * @see net.sf.memoranda.PlanSummary#getLOCModifiedPlan()
+	 */
+	public int getLOCModifiedPlan() {
+		return Integer.parseInt(_el.getAttributeValue("psizeMod"));
+	}
+	
+	/**
+	 * 
+	 * @see net.sf.memoranda.PlanSummary#getLOCReusedPlan()
+	 */
+	public int getLOCReusedPlan() {
+		return Integer.parseInt(_el.getAttributeValue("psizeReused"));
+	}
+	
+	/**
+	 * 
+	 * @see net.sf.memoranda.PlanSummary#getLOCNewPlan()
+	 */
+	public int getLOCNewPlan() {
+		return Integer.parseInt(_el.getAttributeValue("psizeNew"));
+	}
+	
+	/**
+	 * 
+	 * @see net.sf.memoranda.PlanSummary#getTimePlanningPlan()
+	 */
+	public int getTimePlanningPlan() {
+		return Integer.parseInt(_el.getAttributeValue("timePlanning"));
+	}
+	
+	/**
+	 * 
+	 * @see net.sf.memoranda.PlanSummary#getTimeDesignPlan()
+	 */
+	public int getTimeDesignPlan() {
+		return Integer.parseInt(_el.getAttributeValue("timeDesign"));
+	}
+	
+	/**
+	 * 
+	 * @see net.sf.memoranda.PlanSummary#getTimeCodePlan()
+	 */
+	public int getTimeCodePlan() {
+		return Integer.parseInt(_el.getAttributeValue("timeCode"));
+	}
+	
+	/**
+	 * 
+	 * @see net.sf.memoranda.PlanSummary#getTimeCompilePlan()
+	 */
+	public int getTimeCompilePlan() {
+		return Integer.parseInt(_el.getAttributeValue("timeCompile"));
+	}
+	
+	/**
+	 * 
+	 * @see net.sf.memoranda.PlanSummary#getTimeTestPlan()
+	 */
+	public int getTimeTestPlan() {
+		return Integer.parseInt(_el.getAttributeValue("timeTest"));
+	}
+	
+	/**
+	 * 
+	 * @see net.sf.memoranda.PlanSummary#getTimePMPlan()
+	 */
+	public int getTimePMPlan() {
+		return Integer.parseInt(_el.getAttributeValue("timePM"));
+	}
+	
+	/**
+	 * @see net.sf.memoranda.PlanSummary#getLOCperHrToDate()
+	 */
+	public int getLOCperHrToDate(CalendarDate cd) {
+		//NYI
+		return 0;
+	}
+
+	/**
+	 * @see net.sf.memoranda.PlanSummary#getLOCDeletedActual()
+	 */
+	public int getLOCDeletedActual(CalendarDate cd) {
+		// TODO Auto-generated method stub
+		return 0;
```

Comment:
Yes - those methods are planned for implementation after the GUI and after tasks have confirmed to support the necessary features.



On Sat, Feb 13, 2016 at 11:10 PM -0800, "krcarey1" <notifications@github.com<mailto:notifications@github.com>> wrote:


In src/net/sf/memoranda/PlanSummaryImpl.java<https://github.com/cst316/spring16project-Team-El-Paso/pull/27#discussion_r52838793>:

> +     }
> +
> +     /**
> +      * @see net.sf.memoranda.PlanSummary#getLOCperHrToDate()
> +      */
> +     public int getLOCperHrToDate(CalendarDate cd) {
> +             //NYI
> +             return 0;
> +     }
> +
> +     /**
> +      * @see net.sf.memoranda.PlanSummary#getLOCDeletedActual()
> +      */
> +     public int getLOCDeletedActual(CalendarDate cd) {
> +             // TODO Auto-generated method stub
> +             return 0;


Question: Is this and below all still to do?

-
Reply to this email directly or view it on GitHub<https://github.com/cst316/spring16project-Team-El-Paso/pull/27/files#r52838793>.